src/aui/framemanager.cpp

```diff
@@ -1154,13 +1156,15 @@ wxString wxAuiManager::SavePaneInfo(const wxAuiPaneInfo& pane)
     result += wxString::Format(wxT("floatx=%d;"), pane.floating_pos.x);
     result += wxString::Format(wxT("floaty=%d;"), pane.floating_pos.y);
     result += wxString::Format(wxT("floatw=%d;"), pane.floating_size.x);
-    result += wxString::Format(wxT("floath=%d"), pane.floating_size.y);
+    result += wxString::Format(wxT("floath=%d;"), pane.floating_size.y);
+    result += wxString::Format(wxT("cli_floatw=%d;"), pane.floating_client_size.x);
+    result += wxString::Format(wxT("cli_floath=%d"), pane.floating_client_size.y);
```

Comment:
This is pretty minor, but why change the order of words in "floating client size" here? I'd use something like

```suggestion
    result += wxString::Format(wxT("floatw_cli=%d;"), pane.floating_client_size.x);
    result += wxString::Format(wxT("floath_cli=%d"), pane.floating_client_size.y);
```

for these keys.

Reply:
Done.  See https://github.com/wxWidgets/wxWidgets/pull/25483/commits/09466113489fef6757ff32a3ce1211f6f97514aa